### PR TITLE
side barUI

### PR DESCRIPTION
**Pull request by quangchien0806**

### CODE REVIEW

**Comment by quangchien0806** [comment]
[Sidebar UI](https://trello.com/c/bneCHk6Q/56-sidebar-ui)

File: src/components/sideBar/style.scss [hunk 1]
```diff
@@ -0,0 +1,101 @@
+img {
```
Comment (phongnguyenhh1996) [inline src/components/sideBar/style.scss]: add class

File: src/components/sideBar/style.scss [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+  max-width: 100%;
+}
+.all {
+  display: flex;
+}
+.sidebar {
+  background-color: var(--bs-sidebar-bg);
```
Comment (phongnguyenhh1996) [inline src/components/sideBar/style.scss]: define --bs-sidebar-bg or use hex color

File: package.json [hunk 3]
```diff
@@ -12,10 +12,13 @@
     "rc-menu": "^9.7.2",
     "rc-tooltip": "^5.2.2",
     "react": "^18.2.0",
+    "react-custom-scrollbars-2": "^4.5.0",
     "react-dom": "^18.2.0",
     "react-icons": "^4.6.0",
+    "react-indiana-drag-scroll": "^2.2.0",
     "react-redux": "^8.0.5",
     "react-scripts": "5.0.1",
+    "react-select-search": "^4.1.6",
```
Comment (phongnguyenhh1996) [inline package.json:21]: remove unused library

**Review by phongnguyenhh1996 — APPROVED** [review]

**Pull request merged**